Content.Client/Eye/Blinding/BlindOverlay.cs

```diff
@@ -73,6 +80,10 @@ protected override void Draw(in OverlayDrawArgs args)
                 _blindableComponent.GraceFrame = false;
             }
 
+            if (_entityManager.TryGetComponent<ContentEyeComponent>(playerEntity, out var content))
+            {
+                _circleMaskShader?.SetParameter("ZOOM", content.TargetZoom.X);
```

Comment:
TargetZoom is the lerp target; ideally this should be using `EyeComponent`'s `Zoom` directly